tutorials/metrics.md

```diff
@@ -0,0 +1,39 @@
+`/etc/crio/crio.conf`:
+
+```toml
+[crio.metrics]
+enable_metrics = true
+```
+
+The metrics endpoint serves per default on port `9090`. This can be changed via
+the `--metrics-port` command line argument or via the configuration file, too:
+
+```toml
+metrics_port = 9090
+```
+
+If CRI-O runs with enabled metrics, then this can be verified by querying the
+endpoint manually via [curl][1].
+
+```bash
+> curl localhost:9090/metrics
+…
+```
+
+## Available Metrics
+
+Beside the [default golang based metrics][2], CRI-O provides the following additional metrics too:
+
+| Metric Key                             | Possible Labels                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Type    | Purpose                                                                  |
+| -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------- | ------------------------------------------------------------------------ |
+| `crio_operations`                      | every CRI-O RPC name: `Attach`, `ContainerStats`, `ContainerStatus`, `CreateContainer`, `Exec`, `ExecSync`, `ImageFsInfo`, `ImageStatus`, `ListContainerStats`, `ListContainers`, `ListImages`, `ListPodSandbox`, `PodSandboxStatus`, `PortForward`, `PullImage`, `RemoveContainer`, `RemoveImage`, `RemovePodSandbox`, `ReopenContainerLog`, `RunPodSandbox`, `StartContainer`, `Status`, `StopContainer`, `StopPodSandbox`, `UpdateContainerResources`, `UpdateRuntimeConfig`, `Version` | Counter | Cumulative number of CRI-O operations by operation type.                 |
+| `crio_operations_latency_microseconds` | _every CRI-O RPC name_,<br>`network_setup_pod`: CNI pod network setup time,<br>`network_setup_overall`: Overall network setup time                                                                                                                                                                                                                                                                                                                                                         | Summary | Latency in microseconds of CRI-O operations. Split-up by operation type. |
```

Comment:
So I think you're saying here is that every CRI-O RPC name can be used to get metrics for crio_operations, then for the crio_operations_* ones the CRI-O RPC's  plus the other listed items can be used.   If so, then this is fine, although to make it easier to read, just link "Every CRI-O RPC" to a list elsewhere on the page that would look like:

```
CRI-O RPC's: `Attach`, `ContainerStats`, `ContainerStatus`, `CreateContainer`, `Exec`, `ExecSync`, `ImageFsInfo`, `ImageStatus`, `ListContainerStats`, `ListContainers`, `ListImages`, `ListPodSandbox`, `PodSandboxStatus`, `PortForward`, `PullImage`, `RemoveContainer`, `RemoveImage`, `RemovePodSandbox`, `ReopenContainerLog`, `RunPodSandbox`, `StartContainer`, `Status`, `StopContainer`, `StopPodSandbox`, `UpdateContainerResources`, `UpdateRuntimeConfig`, `Version`
```

Or better yet, to a page documenting the RPC's.

Reply:
I added the RPCs below the table to make it easier to read, WDYT?